static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -218,6 +218,42 @@ function isJavascriptSDKTransaction(transaction: TraceTree.Transaction): boolean
   );
 }
 
+function isPageloadTransaction(node: TraceTreeNode<TraceTree.NodeValue>): boolean {
+  return isTransactionNode(node) && node.value['transaction.op'] === 'pageload';
+}
+
+function isSSRTransaction(node: TraceTreeNode<TraceTree.NodeValue>): boolean {
```

Comment:
Idk, maybe we want to name this something like `isServerRequestHandlerTransaction` instead because `op: http.server` doesn't really qualify SSR transactions. It's more generic.